### ISSUE TITLE
Unnecessary equal sign(api.rst line:59,61)

### ISSUE THREAD

**Issue by wang364** [kivy/pyjnius#615]
```
                __javaclass__ == 'java/lang/String'
                __metaclass__ = MetaJavaClass
                __javaconstructor__ == (
```
should be
```
                __javaclass__   = 'java/lang/String'
                __metaclass__ = MetaJavaClass
                __javaconstructor__ = (
```

**Comment by hx2A** [comment]
Thanks for pointing out this mistake. It will be fixed shortly.

**Comment by misl6** [comment]
Fixed via #616
